packages/tooltip/src/uxp-tooltip.css

```diff
@@ -12,97 +12,121 @@ governing permissions and limitations under the License.
 
 /* write uxp style overrides */
 
+/* Workaround for - https://jira.corp.adobe.com/browse/UXP-22211 */
+
 :host {
     --uxp-swc-tooltip-factor: -0.7;
 }
 
-/* Workaround for - https://jira.corp.adobe.com/browse/UXP-21333 */
-#label {
-    margin-bottom: var(
-        --spectrum-tooltip-neutral-text-margin-top,
-        var(--spectrum-global-dimension-static-size-50)
-    );
-}
-
-/* Workarounds for - https://jira.corp.adobe.com/browse/UXP-21118 */
-#tip:after {
-    border-width: calc(
-        -1 * var(--uxp-swc-tooltip-factor) * var(--spectrum-tooltip-tip-height)
-    );
-}
-
-:host([placement*='top']) #tip:after {
-    transform: rotate(45deg);
+:host([placement*='top']) #tooltip #tip {
     margin-top: calc(
         var(--uxp-swc-tooltip-factor) * var(--spectrum-tooltip-tip-height)
     );
 }
 
-:host([placement*='left']) #tip:after {
-    transform: rotate(45deg);
+:host([placement*='left']) #tooltip #tip {
     margin-left: calc(
         var(--uxp-swc-tooltip-factor) * var(--spectrum-tooltip-tip-height)
     );
 }
 
-:host([placement*='right']) #tip:after {
-    transform: rotate(45deg);
-    margin-right: calc(
+:host([placement*='right']) #tooltip #tip {
+    left: calc(
```

Comment:
Trying to understand this particular style rule for placement: right. For left, top, bottom we have added margin-left, margin-top and margin bottom respectively but for right we have added left style. Can you please explain this?